pkg/build/gobuild.go

```diff
@@ -604,8 +623,12 @@ func (g *gobuild) buildOne(ctx context.Context, s string, base v1.Image, platfor
 	}
 
 	cfg = cfg.DeepCopy()
+	cfg.Config.Cmd = nil
 	cfg.Config.Entrypoint = []string{appPath}
-	updatePath(cfg)
+	if platform.OS == "windows" {
+		cfg.Config.Entrypoint = []string{`C:\ko-app\` + appFilename(ref.Path()) + ".exe"}
+	}
```

Comment:
I'm wondering if this could be omitted if the `PATH` for Windows is fixed in the comment below?